lib/web_ui/lib/src/engine/platform_dispatcher.dart

```diff
@@ -771,6 +771,17 @@ class EnginePlatformDispatcher extends ui.PlatformDispatcher {
     scheduleFrameCallback!();
   }
 
+  @override
+  void scheduleWarmUpFrame({required ui.VoidCallback beginFrame, required ui.VoidCallback drawFrame}) {
+    Timer.run(beginFrame);
+    // TODO(yjbanov): technically Flutter flushes microtasks between
+    //                onBeginFrame and onDrawFrame. We don't, which hasn't been
+    //                an issue yet, but eventually we'll have to implement it
+    //                properly. (This is the same to-do as the one in
+    //                `initializeEngineServices` in initialization.dart).
```

Comment:
Does this TODO make sense here? My understanding is that because of the `Timer.run`s the warmUp frame actually is (and always has been) flushing microtasks in between?

That's also what the comment in the other file seems to imply:

> // We use timers here to ensure that microtasks flush in between.

Reply:
You're right! Thank you very much for pointing it out. It made sense in the old code but I forgot to rethink it for the new code.